npm/test/index.test.ts

```diff
@@ -116,3 +116,35 @@ test("renderModules does not call authorisation when dialog box is not present o
 
   keycloakInstance.mockRestore()
 })
+
+test("renderModules should initialize the tna-multi-select-search element on the page", async () => {
+  const keycloakInstance = getKeycloakInstance as jest.Mock
+  keycloakInstance.mockImplementation(() => Promise.resolve(mockKeycloak))
+
+  document.body.innerHTML =
+    `
+    <div class="tna-multi-select-search" data-module="multi-select-search">
+        <div class="tna-multi-select-search" data-module="multi-select-search">
```

Comment:
Is all of this HTML needed for the test, as the initialise for the multi-select is only looking for the presence of the `tna-multi-select-search` class on the page?

Reply:
Yeah, we need it. It checks all the child elements while initialising the `tna-multi-select-search` element.